docs/blueprints/EP036.rst

```diff
@@ -0,0 +1,164 @@
+    | (wildcard match) |         |                  |         |                  |         |                  |         |                  |         |                  |
+    |      of_lldp     |  ---->  |                  |  ---->  |   (exact match   |  ---->  |    (exact match  |  ---->  |      (exact      |  ---->  |    (wildcard)    |
+    |      coloring    |  ---->  |                  |  ---->  | in_port + vlan)  |  ---->  |      in_port)    |  ---->  |       match)     |  ---->  |                  |
+    |                  |         |                  |         |                  |         |                  |         |                  |         |                  |
+    |                  |         |                  |         |                  |         |                  |         |                  |         |                  |
+    |   goto_table: 1  |         |   goto_table: 2  |         |   goto_table: 3  |         |   goto_table: 5  |         |                  |         |                  |
+    +------------------+         +------------------+         +------------------+         +------------------+         +------------------+         +------------------+
+
+New collection, **tables**
+==========================
+
+Initially proposed here is a datastructured representing a linked list (without cycles)
+
+  .. code:: javascript
+
+    "pipeline": {
+      0: {
+        "table_miss_flow": {
+          "priority": 123, "match": {},
+          "instructions": [{"instruction_type": "goto_table", "table_id": 1}]
+        },
+        "napps_table_groups": {"coloring": ["base"],"of_lldp": ["base"]}
+      },
+      1: {
+        "table_miss_flow": {
+          "priority": 123, "match": {},
+          "instructions": [{"instruction_type": "goto_table","table_id": 2}]
+        },
+        "napps_table_groups": {"flow_manager": []}
+      },
+      2: {
+        "table_miss_flow": {
+          "priority": 123, "match": {},
+          "instructions": [{"instruction_type": "goto_table","table_id": 3}]
+        },
+        "napps_table_groups": {"mef_eline": ["evpl"]}
+      },
+      3: {
+        "table_miss_flow": {
+          "priority": 123, "match": {},
+          "instructions": [{"instruction_type": "goto_table","table_id": 5}]
+        },
+        "napps_table_groups": {"mef_eline": ["epl"]}
+      }
+      4: {
+        "table_miss_flow": {},
+        "napps_table_groups": {"INT": []}
+      }
+      5: {
+        "table_miss_flow": {},
+        "napps_table_groups": {}
+      }
+    }
+
+Events
+======
+
+Subscribed
+-----------
+  - **kytos/flow_manager.flow.added**
+  - **kytos/of_core.handshake.completed**
+
+Published
+---------
+
+kytos/multi_table.deploy_table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This event should sent before any NApp sends its first flow to be published.
+

```

Comment:
Please, let's also explain and signal the effort that each NApp that will subscribe to this event needs to document and implement which "table group" it supports. Table group is a new concept being introduced, so it's worth explaining a bit more just so core team members don't get confused (when reviewing or when introducing a new NApp)